.dir-locals.el

```diff
@@ -0,0 +1,2 @@
+((haskell-mode
+  (intero-targets . "stack_linux.yaml")))
```

Comment:
This looks like some editor (emacs?) config file. Is it needed for snap?

Reply:
It is. People using Emacs might find this useful, since we use a special "stack.yaml" file, which breaks `intero` on Linux.